### PR TITLE
[17.0][MIG] fieldservice_sale_stock: Migration to 17.0

### PR DESCRIPTION
**Pull request by imlopes**

### CODE REVIEW

File: fieldservice_sale_stock/models/sale_order.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def prepare_fsm_values_for_stock_picking(self, fsm_order):
     def _link_pickings_to_fsm(self):
         for rec in self:
             # TODO: We may want to split the picking to have one picking
-            # per FSM order
+            #  per FSM order
```
Comment (yankinmax) [inline fieldservice_sale_stock/models/sale_order.py]: this is not needed, typo?

**Review by yankinmax — APPROVED** [review]

File: fieldservice_sale_stock/readme/INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+To install Field Service and have the mapping features, you need to
+install GeoEngine.
+
+Please refer to the installation instructions available at:
+<https://github.com/OCA/geospatial/tree/12.0/base_geoengine>
```
Comment (gurneyalex) [inline fieldservice_sale_stock/readme/INSTALL.md]: I think this is an unrelated instruction that has been there since 12.0 

Can you remove the file from this addon?

**Review by gurneyalex — APPROVED** [review]

**Review by max3903 — APPROVED** [review]
